Dockerfile

```diff
@@ -1,18 +1,23 @@
-FROM civisanalytics/datascience-python:6.5.1
-MAINTAINER support@civisanalytics.com
+# Default value provided here; will be overridden at build time.
+# Unfortunately Dockerhub isn't as flexible as CircleCi or AWS Codebuild, so we have to hardcode this value here.
+# So if you in the future need to update this value, make sure you also edit the value in .ds_python_version.
+# These values should be kept in sync.
+ARG DS_PYTHON_IMG_VERSION=7.0.0
```

Comment:
Not sure if I'm overthinking this, but my hunch is that we'd still want a little assertion in CircleCI to assert that this default value equals the tag in `.ds_python_version`, in order to avoid the following situation:

* The next maintainer sees `.ds_python_version` and updates the tag in it. If they happen to not look at `Dockerfile` (or not do so carefully), they may not update this default value here in `Dockefile`.
* Then the code change gets approved and released.
* A new image with the new tag gets built on AWS Codebuild, but DockerHub would raise an error for trying to build for a tag that has already existed...

Reply:
i talked with matt brennen about updating our AWS CodeBuild configuration to push to dockerhub instead of having docekrhub build the image.

so sit tight until we get that configuration in place.